packages/web/package.json

```
@@ -52,7 +52,7 @@
     "@babel/preset-typescript": "^7.21.0",
     "@bgotink/playwright-coverage": "^0.3.2",
     "@cdssnc/gcds-fonts": "^0.2.2",
```

could we use the new fonts package in this PR?
```suggestion
    "@gcds-core/fonts": "^1.0.0",
```